packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,73 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop                      | Type                                                       | Default     | Description                                                                                                                                                                                       | Required |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| value                     | `number`                                                   | `undefined` | The value to be rendered within the component                                                                                                                                                     | ✅       |
+| variant                   | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                                                                                                                                          |          |
+| fontWeight                | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                                                                                                                                           |          |
+| showSuffix                | `none`, `decimals` , `humanise`                            | `decimals`  | The type of affix to be displayed with the currency value, (none, decimals, and humanise) that respectively indicate no affix, the number of decimal places, and the scale of the currency value. |          |
+| size                      | `3xlarge`, `2xlarge`, `xlarge`, `large`, `medium`, `small` | `3xlarge`   | The size of the amount to be rendered                                                                                                                                                             |          |
+| isSuffixPrefixHighlighted | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small                                                                                                                    |          |
```

Comment:
We need to changes this to:
```suggestion
| isAffixHighlighted | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small                                                                                                                    |          |
```

Reply:
@chaitanyadeorukhkar when this value will be true, the affix won't be highlighted, it is the value which will get highlighted.

Reply:
Yes, it is incorrect and open to be replaced with a better name.

Reply:
**isHighlighted=true** 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/63398163/220062152-dd4c7351-e062-4dd0-9594-26302f378175.png">
**isHighlighted=false** 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/63398163/220062244-a5d1ce15-f222-4714-b5a6-4aac23f635a4.png">

Reply:
Can it be like this:

`isAffixHighlighted=true`
![](https://user-images.githubusercontent.com/63398163/220062244-a5d1ce15-f222-4714-b5a6-4aac23f635a4.png)

`isAffixHighlighted=false`
![](https://user-images.githubusercontent.com/63398163/220062152-dd4c7351-e062-4dd0-9594-26302f378175.png)

Reply:
This is really taking a test of my English grammar 😅 

How about `isAffixDeemphasized`?

Reply:
Why go in the negation though? `isAffixHighlighted` would still work

Reply:
Because it isn't actually "highlighted" (in terms of the language), it matches the other content (the number part - both have same size, weight etc.) 😅

Reply:
Yeah, Makes sense. How about `isAffixSubtle`? Seems a bit simpler to write than `isAffixDeemphasized`

Reply:
Should we call it `isCurrencyHighlighted` instead? I understand that it also changes emphasis on decimal number but maybe just to keep the name simple?

TIL `Affix` 🙈

Reply:
Actually currency isn't "highlighted" per se, just the other things are shrinked and faded 😅 

So far these two seems like best options to me:
- `isAffixDeemphasized`
- `isAffixSubtle`

Reply:
let's go with this `isAffixSubtle` ?

Reply:
+1 to `isAffixSubtle`

Reply:
Lets fix `isAffixSubtle` then 👍